lib/app/utils.dart

```diff
@@ -32,8 +32,12 @@ String formatFileSize(int bytes) {
   return '${(bytes / power).toStringAsFixed(index == 0 ? 0 : 1)} ${units[index]}';
 }
 
-/// Converts a DateTime to a string.
-String dateTimeToString(DateTime dt) => DateFormat.yMMMd().format(dt);
+/// Converts a [DateTime] to a [String].
+String dateTimeToString(DateTime dt) => DateFormat.MMMd().format(dt);
```

Comment:
Move to using `Instant` and use a formatter of `time_machine` in concrete modules.

Reply:
As issue #117 already entails revising how `DateTimes` are handled throughout the project, I'd argue we rewrite this at a later `Instant` (sorry for the pun).